src/Avalonia.Themes.Default/ComboBox.xaml

```diff
@@ -58,7 +58,16 @@
   <Style Selector="ComboBox:pointerover /template/ Border#border">
     <Setter Property="BorderBrush" Value="{DynamicResource ThemeBorderHighBrush}"/>
   </Style>
-    <Style Selector="ComboBox:disabled /template/ Border#border">
-        <Setter Property="Opacity" Value="{DynamicResource ThemeDisabledOpacity}" />
-    </Style>
+  <Style Selector="ComboBox:disabled /template/ Border#border">
+    <Setter Property="Opacity" Value="{DynamicResource ThemeDisabledOpacity}" />
+  </Style>
+  <Style Selector="ComboBox[SelectedIndex=-1] /template/ ContentControl">
```

Comment:
Hmm, not 100% convinced personally that setting the `ContentControl` content like this is the best way to go about things, in particular because I think we may need to change `ContentControl.Content` to be a direct property instead of a styled property in future.

Do you know by any chance how they do this in WPF? Just so we can compare approaches.

Reply:
A `MinHeight` also works.

This is WPF's default:

```
 <Setter Property="MinWidth"
          Value="120" />
  <Setter Property="MinHeight"
          Value="20" />
```

Reply:
A MinHeight would solve the problem as well, it is also what MahApps does. But I have no idea how this reacts to font scaling, e.g. when ComboxItem text size uses more height than ComboBox minheight property it likely would still resize.

Just let me know what you prefer so I can change it

Reply:
ComboBox items can have non-uniform heights. You can see an example of that in our control catalog. It's not guaranteed that there would be any text at all. So the minimal height is only needed for ComboBox to be able to accommodate the expander arrow and not look ugly. So it should be fine to use WPF's default of MinHeight=20.